app/controllers/participate/login.js

```diff
@@ -15,7 +15,10 @@ export default Ember.Controller.extend({
     authenticate(attrs) {
       this.get('session')
         .authenticate('authenticator:jam-jwt', attrs)
-        .then(() => this.transitionToRoute('participate.survey.consent'))
+        .then(() => {
+          this.set('studyId', attrs['password']);
```

Comment:
Ok, I can also save the studyId on the session -- right now it's just available as the user's account password